apis/python/src/tiledbsoma/__init__.py

```diff
@@ -93,6 +93,49 @@
 # crucial that we include a separator (e.g. "Classes and functions") to make an entry in the
 # readthedocs table of contents.
 
+import ctypes
+import os
+import sys
+
+
+# Load native libraries. On wheel builds, we may have a shared library
+# already linked. In this case, we can import directly
+try:
+    from . import pytiledbsoma as clib
+
+    del clib
+except ImportError:
+    if os.name == "nt":
+        lib_name = "tiledb.dll"
+    elif sys.platform == "darwin":
+        lib_name = "libtiledb.dylib"
+    else:
+        lib_name = "libtiledb.so"
+
+    try:
+        # Try loading the bundled native library.
+        lib_dir = os.path.dirname(os.path.abspath(__file__))
+        ctypes.CDLL(os.path.join(lib_dir, lib_name), mode=ctypes.RTLD_GLOBAL)
+    except OSError:
+        # Otherwise try loading by name only.
+        ctypes.CDLL(lib_name)
+
+    if os.name == "nt":
+        lib_name = "tiledbsoma.dll"
+    elif sys.platform == "darwin":
+        lib_name = "libtiledbsoma.dylib"
+    else:
+        lib_name = "libtiledbsoma.so"
+
+    try:
+        # Try loading the bundled native library.
+        lib_dir = os.path.dirname(os.path.abspath(__file__))
+        ctypes.CDLL(os.path.join(lib_dir, "libtiledb.so"), mode=ctypes.RTLD_GLOBAL)
+        ctypes.CDLL(os.path.join(lib_dir, lib_name))
+    except OSError:
+        # Otherwise try loading by name only.
+        ctypes.CDLL(lib_name)
```

Comment:
It looks like this code was duplicated

Reply:
I removed the extraneous line. The top block loads tiledb where this bottom block loads tiledbsoma.

Reply:
> The top block loads tiledb where this bottom block loads tiledbsoma.

Sorry, my bad. I stared at this forever trying to figure out what was different, and I obviously missed that `lib_name` was the only thing that changed